test/integration/README

```diff
@@ -1,11 +1,5 @@
 To run integration tests:
-1. Replace 'describe.skip' with 'describe' in test/integration/integration.js
-2. Create a file at ~/.ripple_wallet containing a JSON object like this with
-   your own test address and secret.
-    {
-      "test": {
-        "address": "r3GgMwvgvP8h4yVWvjH1dPZNvC37TjzBBE",
-        "secret": "shsWGZcmZz6YsWWmcnpfr6fLTdtFV"
-      }
-    }
-3. Run "mocha test/integration/integration.js"
+1. Run rippled-standalone node.
+  * Either with docker `docker run -p 6006:6006 -it natenichols/rippled-standalone:latest`
```

Comment:
This should mention something about how the docker method is very easy (or do you think that's implied enough?)

Reply:
I was thinking that was implied. I can add a note.